JamGame/Assets/Scripts/Employee/EmployeeImpl.cs

```diff
@@ -168,12 +168,14 @@ private void Update()
                         currentlySatisfyingNeed = null;
 
                         targetNeedProvider.Release();
-                        // TODO refactor
+
+                        // TODO: Remove it when employee serialization will developed
```

Comment:
```suggestion
                        // TODO: Remove it when employee serialization will be implemented (#121)
```
please reference issues by their number everytime.

Reply:
the same in other comments